web/viewer.css

```diff
@@ -1509,7 +1509,8 @@ html[dir='rtl'] #documentPropertiesOverlay .row > * {
   display: none;
 }
 
-@media screen and (min-resolution: 2dppx) {
+@media screen and (min-resolution: 2dppx),
+       screen and (-webkit-min-device-pixel-ratio: 2) {
```

Comment:
.. Or modification/improvement of the preprocessor is required along with this PR